src/components/Main.vue

```diff
@@ -122,18 +144,60 @@ export default {
 
     computed: {
         circlePoints () {
```

Comment:
The `circlePoints` computed property should return circle points sorted in logical order, because `linePoints` depends on that. At the moment the final result is somewhat unexpected, because lines are not correctly drawn. You can check this out by running the generator on the unmodified code, and then comparing its output with your branch, when the center x/y displacement is 0/0 (should result in the same image).